cli/impl/src/main/assembly/distribution.xml

```diff
@@ -0,0 +1,55 @@
+<!--
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.1.1"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.1.1 https://maven.apache.org/xsd/assembly-2.1.1.xsd">
+    <id>distribution</id>
+    <includeBaseDirectory>true</includeBaseDirectory>
+
+    <formats>
+        <format>zip</format>
+    </formats>
+    <fileSets>
+        <fileSet>
+            <directory>${project.build.directory}/libs</directory>
+            <outputDirectory>/lib/libs</outputDirectory>
+            <includes>
+                <include>*.jar</include>
+            </includes>
+        </fileSet>
+    </fileSets>
+    <files>
+        <file>
+            <source>${project.build.directory}/helidon.jar</source>
+            <outputDirectory>/lib</outputDirectory>
+        </file>
+        <file>
+            <source>${project.basedir}/src/main/assembly/helidon.bat</source>
+            <outputDirectory>/bin</outputDirectory>
+        </file>
+        <file>
+            <source>${project.basedir}/src/main/assembly/helidon</source>
```

Comment:
How about permission ? Is the script executable when unpacking the distribution.

Reply:
This is taken care of during installation process. If you just unpack it and use it right away, you need to `chmod +x helidon`

Reply:
This shouldn't be required. Permissions can be set and preserved when unpacking.